GET /metrics panic

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
`curl http://localhost:9567/metrics`

```
# ./juicefs mount redis://localhost/1 ~/jfs
2021/02/22 03:51:19.283571 juicefs[30646] <INFO>: Meta address: redis://localhost/1
2021/02/22 03:51:19.284559 juicefs[30646] <WARNING>: AOF is not enabled, you may lose data if Redis is not shutdown properly.
2021/02/22 03:51:19.284939 juicefs[30646] <INFO>: Data use localhost:9000/juicefs/
2021/02/22 03:51:19.284959 juicefs[30646] <INFO>: Mounting volume juicefs at /root/jfs ...
2021/02/22 03:51:19.284986 juicefs[30646] <INFO>: Cache: /var/jfsCache/10fdadd2-f87f-435f-bd31-7a81ada8c31d capacity: 1024 MB
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x2011dc9]

goroutine 134 [running]:
github.com/minio/minio/cmd.(*NotificationSys).ServerInfo(0x0, 0x1, 0xc001796c90, 0x0)
	/root/go/pkg/mod/github.com/minio/minio@v0.0.0-20210206053228-97fe57bba92c/cmd/notification.go:1222 +0x29
github.com/minio/minio/cmd.GetPeerOnlineCount(0x2dc8800, 0x2e10e00)
	/root/go/pkg/mod/github.com/minio/minio@v0.0.0-20210206053228-97fe57bba92c/cmd/notification.go:1306 +0x3f
github.com/minio/minio/cmd.nodeHealthMetricsPrometheus(0xc000700de0)
	/root/go/pkg/mod/github.com/minio/minio@v0.0.0-20210206053228-97fe57bba92c/cmd/metrics.go:108 +0x26
github.com/minio/minio/cmd.(*minioCollector).Collect(0xc000724ae0, 0xc000700de0)
	/root/go/pkg/mod/github.com/minio/minio@v0.0.0-20210206053228-97fe57bba92c/cmd/metrics.go:98 +0xe5
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
	/root/go/pkg/mod/github.com/prometheus/client_golang@v1.9.0/prometheus/registry.go:446 +0x1a2
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/root/go/pkg/mod/github.com/prometheus/client_golang@v1.9.0/prometheus/registry.go:538 +0xe8e
```

**What you expected to happen**: No panic

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `./juicefs --version`) or Hadoop Java SDK version: `juicefs version 0.10.0-49 (2021-02-20 85fcdb6)`
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Object storage (cloud provider and region):
- Redis info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to Redis, JuiceFS to object storage):
- Others:

--- a/go.mod
+++ b/go.mod
@@ -52,3 +52,5 @@ require (
 	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
 	google.golang.org/api v0.5.0
 )
+
+replace github.com/minio/minio v0.0.0-20210206053228-97fe57bba92c => github.com/juicedata/minio v0.0.0-20210222051636-e7cabdf948f4

